questions/multiple_whys_in_sprint_goal.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: "Multiple Whys in Sprint Goal"
+parent: Katas
+---
+The Scrum Guide states: “The whole Scrum Team then collaborates to define a Sprint Goal that communicates why the Sprint is valuable to stakeholders”.
+
+Can a single Sprint Goal contain several points of value to stakeholders? 
+Example: “Create a login page so that users can securely enter the platform AND learn the basic information about our product (as we don’t have a homepage yet)”.
```

Comment:
```suggestion
The Scrum Guide states: "The whole Scrum Team then collaborates to define a Sprint Goal that communicates why the Sprint is valuable to stakeholders".

Can a single Sprint Goal contain several points of value to stakeholders? 
Example: "Create a login page so that users can securely enter the platform AND learn the basic information about our product (as we don’t have a homepage yet)".
```

Copy/paste sometimes adds strange double-quotes (you can see they are different than the regular ones). This _might_ cause issues somewhere so I think it's better to use regular ones ;-)